projects/1.16/assets/dynamic-trees-byg/dtbyg/lang/zh_cn.json

```
@@ -0,0 +1,208 @@
+    "block.dtbyg.ether_branch": "以太树",
+    "block.dtbyg.nightshade_branch": "夜影树",
+    "block.dtbyg.oddity_cactus_branch": "子实体仙人掌",
+  
+    "species.dtbyg.maple": "枫树",
+    "species.dtbyg.maple_twiglet": "枫树树枝",
+    "species.dtbyg.red_maple": "红色枫树",
+    "species.dtbyg.silver_maple": "银色枫树",
+    "species.dtbyg.ancient_silver_maple": "古老银色枫树",
+    "species.dtbyg.blue_enchanted": "蓝色魔法树",
+    "species.dtbyg.green_enchanted": "绿色魔法树",
+    "species.dtbyg.blue_enchanted_poplar": "蓝色魔法白杨树",
+    "species.dtbyg.green_enchanted_poplar": "绿色魔法白杨树",
+    "species.dtbyg.poplar": "白杨树",
+    "species.dtbyg.yellow_poplar": "黄杨树",
+    "species.dtbyg.orange_spruce": "橙色云杉树",
+    "species.dtbyg.red_spruce": "红色云杉树",
+    "species.dtbyg.yellow_spruce": "黄色云杉树",
+    "species.dtbyg.blue_spruce": "蓝色云杉树",
+    "species.dtbyg.tall_spruce": "高大的云杉树",
+    "species.dtbyg.tall_orange_spruce": "高大的橙色云杉树",
+    "species.dtbyg.tall_red_spruce": "高大的红色云杉树",
+    "species.dtbyg.tall_yellow_spruce": "高大的黄色云杉树",
+    "species.dtbyg.tall_blue_spruce": "高大的蓝色云杉树",
+    "species.dtbyg.mega_orange_spruce": "巨大的橙色云杉树",
+    "species.dtbyg.mega_red_spruce": "巨大的红色云杉树",
+    "species.dtbyg.mega_yellow_spruce": "巨大的黄色云杉树",
+    "species.dtbyg.mega_blue_spruce": "巨大的蓝色云杉树",
+    "species.dtbyg.giga_spruce": "超大的云杉",
+    "species.dtbyg.giga_orange_spruce": "超大的橙色云杉",
+    "species.dtbyg.giga_red_spruce": "超大的红色云杉",
+    "species.dtbyg.giga_yellow_spruce": "超大的黄色云杉",
+    "species.dtbyg.giga_blue_spruce": "超大的蓝色云杉",
+    "species.dtbyg.northern_spruce": "北方云杉",
+    "species.dtbyg.brown_birch": "棕色桦树",
+    "species.dtbyg.orange_birch": "橙色桦树",
+    "species.dtbyg.red_birch": "红色桦树",
+    "species.dtbyg.yellow_birch": "黄色桦树",
+    "species.dtbyg.tall_brown_birch": "高大的棕色桦树",
+    "species.dtbyg.tall_orange_birch": "高大的橙色桦树",
+    "species.dtbyg.tall_red_birch": "高大的红色桦树",
+    "species.dtbyg.tall_yellow_birch": "高大的黄色桦树",
+    "species.dtbyg.brown_oak": "棕色橡树",
+    "species.dtbyg.mega_brown_oak": "巨大的棕色橡树",
+    "species.dtbyg.orange_oak": "橙色橡树",
+    "species.dtbyg.mega_orange_oak": "巨大的橙色橡树",
+    "species.dtbyg.red_oak": "红色橡树",
+    "species.dtbyg.mega_red_oak": "巨大的红色橡树",
+    "species.dtbyg.tapered_oak": "锥型橡树",
+    "species.dtbyg.brown_tapered_oak": "棕色锥型橡树",
+    "species.dtbyg.orange_tapered_oak": "橙色锥型橡树",
+    "species.dtbyg.red_tapered_oak": "红色锥型橡树",
+    "species.dtbyg.prairie_oak": "草原橡树",
+    "species.dtbyg.mega_prairie_oak": "巨大的草原橡树",
+    "species.dtbyg.oak_twiglet": "橡树树枝",
+    "species.dtbyg.red_oak_twiglet": "红色橡树树枝",
+    "species.dtbyg.canopy_oak": "冠层橡树",
+    "species.dtbyg.ebony_oak": "乌木橡树",
+    "species.dtbyg.joshua": "约书亚",
+    "species.dtbyg.orchard": "果树",
+    "species.dtbyg.mega_orchard": "巨大的果树",
+    "species.dtbyg.tall_orchard": "高大的果树",
+    "species.dtbyg.dark_oak_twiglet": "深色橡树树枝",
+    "species.dtbyg.canopy_dark_oak": "冠层深色橡树",
+    "species.dtbyg.ancient_dark_oak": "古老深色橡树",
+    "species.dtbyg.aspen": "山杨",

```

```suggestion
    "species.dtbyg.aspen": "白杨",
```
下同，也要验证一下语境里要不要“木”这个字